platform/commonUI/general/src/controllers/TimeRangeController.js

```diff
@@ -243,36 +211,12 @@ define(
                 updateTicks();
             }
 
-            function updateStartFromText(value) {
-                try {
-                    updateOuterStart(parseTimestamp(value));
-                    updateBoundsTextForProperty($scope.ngModel, 'end');
-                    $scope.boundsModel.startValid = true;
-                } catch (e) {
-                    $scope.boundsModel.startValid = false;
-                    return;
-                }
-            }
+            function updateFormat(key) {
+                formatter = formatService.getFormat(key) ||
```

Comment:
Hrm, wondering if it's wise to fallback to utc when a requested format isn't available-- would it be better to throw an error-- perhaps at the level of the format service?  

Also, at the level of the time conductor, it should always be getting the format set by the selected domain, correct?  So it shouldn't need to have a separate "default format".